db/migrate/20170112194141_change_name_to_friendly_name.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeNameToFriendlyName < ActiveRecord::Migration
+  def change
+    rename_column :service_providers, :name, :friendly_name
```

Comment:
given that no existing dashboard instance should have any users or service provider records in it, I think it is safe to say we just deploy and suffer the downtime blip if it happens.